runtime/caml/tsan.h

```diff
@@ -40,6 +40,24 @@
 #  endif
 #endif
 
+/* TSAN annotates a release operation on encountering ANNOTATE_HAPPENS_BEFORE
+ * and similarly an acquire operation on encountering ANNOTATE_HAPPENS_AFTER */
```

Comment:
```suggestion
/* TSan records a release operation on encountering ANNOTATE_HAPPENS_BEFORE
 * and similarly an acquire operation on encountering ANNOTATE_HAPPENS_AFTER.
   These annotations are used to eliminate false positives. */
```

“TSan records an operation” seems more accurate to me than “TSan annotates an operation”. Also minor: TSAN -> TSan.